stdlib/json/encode_test.go

```diff
@@ -25,8 +25,9 @@ o = {
     d: false,
     e: /.*/,
 	f: 2019-08-14T10:03:12Z,
+    g: [1: "hi", 2: "there"]
```

Comment:
Nit: We could add a dictionary with string keys to the test, just to guard against someone changing `values.Display` to format strings with `"` around it to distinguish `2` from `"2"`.